brainscore_language/__init__.py

```diff
@@ -22,6 +22,9 @@ def load_dataset(identifier: str) -> Union[DataAssembly, Any]:
     # imports to load plugins until plugin system is implemented
     from brainscore_language.plugins.wikitext_next_word_prediction import data
     from brainscore_language.plugins.futrell2018 import data
+    from brainscore_language.plugins.pereira2018 import data
```

Comment:
@gretatuckute are you referring to liking something about the `pereira2018` data import specifically or more generally about importing the plugins? (If the latter, would be good to know for building out plugin system.)

Reply:
no, not pereira2018 specifically, but the structuring of benchmarks into their respective papers instead of e.g., Schrimpf2021 with several different benchmarks. I assume we are going to change the plugins folder names here though?

Reply:
👍 thanks for clarifying. 

Yes, once this PR is merged, I'll restructure the directories so that e.g. `pereira2018` data would be at `brainscore_language/data/pereira2018`.

Reply:
@aalok-sathe the imports are just a placeholder while the plug-in system is being built.

All plug-ins will be run in an environment created for the plugins specified.

Reply:
All these imports will actually go away (aka be magically available) with the plugin system that @kvfairchild is implementing